### DISCUSSION TITLE
model results randomly feasible and infeasible

### DISCUSSION THREAD

**Discussion by disonvon** [google/or-tools#4309] [CP-SAT questions] [answered]
**What version of OR-Tools and what language are you using?**
Version: 9.9.3963
Language: C++/Java/Python/C#
Python
**Which solver are you using (e.g. CP-SAT, Routing Solver, GLOP, BOP, Gurobi)**
CP-SAT
**What operating system (Linux, Windows, ...) and version?**
Mac Os 13.0 (22A380)
**What did you do?**
attached two model files are generated from two independent run with same input, but one is feasible and another is not. the two files are the same but the variable or the constraint order

**What did you expect to see**
the two results should be aligned and feasible
[test_feasible.pd.txt](https://github.com/user-attachments/files/16186336/test_feasible.pd.txt)
[test_infeasible.pd.txt](https://github.com/user-attachments/files/16186338/test_infeasible.pd.txt)
<img width="1158" alt="image" src="https://github.com/user-attachments/assets/6831f8e3-4047-4ff5-9020-abc79d64af55">

**Comment by lperron** [comment]
Can you try with 9.10?

**Comment by disonvon** [comment]
9.10 also tried, perform the same behavior like 9.9

**Comment by lperron** [comment]
are you sure the models are equivalent ? you change switching one interval between no_overlaps

**Comment by lperron** [comment, marked as answer]
I do not believe this is a bug, Moving to discussions.

**Reply by disonvon** [reply]
it turns out that it was a bug at my side. Thanks for your time.
